### PR TITLE
Update ActionBar template & add link styles

### PR DESCRIPTION
**Pull request by yatriks**
A future pull request will remove the unused `.breadcrumbs` styles, as they're currently in use in production.

_New template:_
image

### CODE REVIEW

File: scss/_adk-layout.scss [hunk 1]
```diff
@@ -13,6 +13,13 @@
   padding: $space-s 0;
   background-color: $white;
   border-bottom: $global-separator;
+  .rightLinks a {
```
Comment (expandrew) [inline scss/_adk-layout.scss]: Do these come with hover styles? I assume they do, since they should just inherit regular `<a>` styles, right?

Comment (yatriks) [inline scss/_adk-layout.scss]: You would think so, but they actually do not – I remember meaning to add hover styles and I forgot...great catch, thanks, brb.

Comment (expandrew) [inline scss/_adk-layout.scss]: Maybe we need `a.blue` or `a.{color}` classes that have hovers on all of them - I feel like there should be a consistent behavior across all links, regardless of whether they have a color class applied

**Comment by yatriks** [comment]
@expandrew I believe `.menu` does exactly this – going to close this PR and resubmit with an updated template using `.menu`

**Pull request closed without merging**